test/functional/wallet_bumpfee.py

```diff
@@ -333,6 +358,7 @@ def test_maxtxfee_fails(self, rbf_node, dest_address):
     assert_raises_rpc_error(-4, "Unable to create transaction. Fee exceeds maximum configured by -maxtxfee", rbf_node.bumpfee, rbfid)
     self.restart_node(1, self.extra_args[1])
     rbf_node.walletpassphrase(WALLET_PASSPHRASE, WALLET_PASSPHRASE_TIMEOUT)
+    self.connect_nodes(1, 0)
 
```

Comment:
I think I'd prefer to see `clear_mempool()` here, on each test end.

Reply:
thx, done